app/parse.py

```diff
@@ -8,5 +13,24 @@ class Course:
     duration: str
 
 
+def get_number_from_months_text(text: str) -> str:
+    return text.split(" ")[0][:-1]
```

Comment:
The function `get_number_from_months_text` assumes that the text will always be in the format 'X months'. If the format changes or if there are other characters, this could lead to incorrect parsing. Consider adding error handling or validation to ensure the text is in the expected format.